src/org/kernelci/build/Kernel.groovy

```diff
@@ -1,66 +0,0 @@
-/*
```

Comment:
The `bisect.jpl` and `build-trigger.jpl` jobs also import this file.  So please don't delete it yet, but update the other jobs accordingly to use the new Python function and drop the import, then it can be deleted.

Reply:
I checked other files for usage of  `downloadTarball` function as it was the only one that was provided by `src/org/kernelci/build/Kernel.groovy`. I missed `bisect.jpl` and `build-trigger.jpl` as these were just importing the `Kernel.groovy`, but not using any of its code.
I removed the unnecessary imports from these files.

Reply:
Oh I see, they were stale imports.  Then that's easier, nice.